packages/components/psammead-most-read/src/Rank/index.test.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { latin, arabic } from '@bbc/gel-foundations/scripts';
+import MostReadRank from '.';
+
+describe('MostReadRank', () => {
+  shouldMatchSnapshot(
+    'should render ltr correctly',
+    <MostReadRank
+      service="news"
+      script={latin}
+      listIndex={1}
+      numberOfItems={5}
+      dir="ltr"
+    />,
+  );
+
+  shouldMatchSnapshot(
+    'should render ltr with double digits correctly',
+    <MostReadRank
+      service="news"
+      script={latin}
+      listIndex={1}
+      numberOfItems={5}
```

Comment:
```suggestion
      numberOfItems={10}
```

Reply:
🙈

Reply:
This will cause a snapshot update to be required too unfortunately.

Reply:
just updating snapshots

![](https://media.giphy.com/media/toXKzaJP3WIgM/giphy.gif)

Reply:
🎉